Tests/appstoreconnect-cliTests/Models/Bundle+Tests.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+extension Bundle {
+    static let tests: Bundle =
+        Bundle(url: URL(fileURLWithPath: #file)
+            .deletingLastPathComponent()
+            .appendingPathComponent("Fixtures.bundle"))!
```

Comment:
I was trying to use something like 
```
private final class BundleTag {}

Bundle(for: BundleTag.self).url(forResource: named, withExtension: "json", subdirectory: "Fixtures.bundle")
```
But couldn't make it work, it keeps telling me can't find bundle 😢